src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -21,6 +21,37 @@ public virtual DirectoryInfo Create(string path)
 
         #region UniversalTests
 
+        [Fact]
+        public void FileNameIsToString_NotFullPath()
+        {
+            // We're checking that we're maintaining the original path
+            RemoteInvoke(() =>
+            {
+                Environment.CurrentDirectory = TestDirectory;
+                string subdir = Path.GetRandomFileName();
+                DirectoryInfo info = Create(subdir);
+                Assert.Equal(subdir, info.ToString());
+            }).Dispose();
+        }
+
+        [Fact]
+        public void FileNameIsToString_FullPath()
+        {
+            string subdir = Path.GetRandomFileName();
+            string fullPath = Path.Join(TestDirectory, subdir);
```

Comment:
This test needs to be separated out into a netcoreapp file, as this API doesn't exist on netfx.

Reply:
Yeah, spaced on this one- thanks.